doc/source/conf.py

```diff
@@ -17,11 +17,47 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
+import os
+import shutil
 import subprocess
 
 # It is assumed that you run sphinx from the virtual environment which
 # includes pycompwa
 
+# -- Copy example notebooks ---------------------------------------------------
+print("Copy example notebook files")
+PATH_SOURCE = "../../examples"
+PATH_TARGET = "usage"
+EXAMPLE_FOLDERS_TO_COPY = [
+    "tools",
+    "workflow",
+]
+IGNORED_PATTERNS = shutil.ignore_patterns(
+    "*/.ipynb_checkpoints/*",
+)
+for root, dirs, _ in os.walk(PATH_TARGET):
+    for directory in dirs:
+        path = os.path.join(root, directory)
+        print("  remove directory", path)
+        shutil.rmtree(path)
+for root, dirs, _ in os.walk(PATH_SOURCE):
+    for directory in dirs:
+        if directory not in EXAMPLE_FOLDERS_TO_COPY:
+            continue
+        path_from = os.path.join(root, directory)
+        path_to = os.path.join(PATH_TARGET, directory)
+        print("  copy", path_from, "to", path_to)
+        shutil.copytree(
+            path_from, path_to,
+            symlinks=True,
+            ignore=IGNORED_PATTERNS,
+        )
+shutil.copyfile(
+    os.path.join(PATH_SOURCE, "particle_list.xml"),
+    os.path.join(PATH_TARGET, "particle_list.xml"),
+    follow_symlinks=True,
+)
```

Comment:
Can't you simplify this to
```python
shutil.copytree(
   PATH_SOURCE, PATH_TARGET, 
   ignore=ignore_except('ipynb'))
```
It might also be better to just just specifically name the sources to avoid trouble if new files are added etc

Reply:
* Tbh, this whole `ignore=IGNORED_PATTERNS` doesn't work :(
* I do want exceptions to be raised by Sphinx if notebooks are not implemented in the documentation.
* It's currently not the case, but you will also want non-`ipynb` files to be copied if they are required by those notebooks.

Reply:
Ok